core/src/main/java/com/oath/oak/OakMap.java

```diff
@@ -882,7 +882,7 @@ public int size() {
         }
     }
 
-    static final class KeyBufferSet<K, V> extends AbstractSet<OakRBuffer> {
+    static final class KeyBufferSet<K, V> extends AbstractSet<OakDetachedBuffer> {
```

Comment:
This is a big change to catch something by eye. Do we have some tests that try to write to read-only "detached" buffer?

Reply:
Trying to write? It does not have a write API.